src/App/Routes/ContributionDetailsPage/View.tsx

```diff
@@ -30,6 +31,9 @@ const ContributionDetailsPage: FC<Props> = ({
   contributorId,
   hasAppliedToContribution = false,
 }) => {
+  if (!contribution) {
```

Comment:
I would avoid doing this since the props define the `contribution` as non optional, I would rather go there https://github.com/onlydustxyz/marketplace-frontend/blob/main/src/App/Routes/ContributionDetailsPage/index.tsx#L88

Reply:
Ok! For some reason I some weird error when trying there, but will double check